trading_calendars/us_holidays.py

```diff
@@ -92,10 +94,11 @@ def july_5th_holiday_observance(datetime_index):
     days_of_week=(MONDAY, TUESDAY, THURSDAY),
     start_date=Timestamp("1995-01-01"),
 )
-FridayAfterIndependenceDayExcept2013 = Holiday(
+FridayAfterIndependenceDayExcept2013and2019 = Holiday(
```

Comment:
Not sure if the policy has changed, and this should be `FridayAfterIndependenceDay2013AndAfter`.

Reply:
I don't know if this is true in the future, it only seems published to 2021: https://www.nyse.com/markets/hours-calendars